tests/testthat/test-plot_step_ahead_model_output.R

```diff
@@ -0,0 +1,231 @@
+# Data Preparation
+projection_path <- system.file("example_round1.csv", package = "hubVis")
+projection_data0 <- read.csv(projection_path, stringsAsFactors = FALSE)
+projection_data <- dplyr::mutate(
+  projection_data0, target_date = as.Date(origin_date) + (horizon * 7) - 1)
+projection_data_A_us <- dplyr::filter(projection_data,
+                                      scenario_id == "A-2021-03-05",
+                                      location == "US")
+
+d_proj = rbind(projection_data_A_us,
+               dplyr::mutate(projection_data_A_us,
+                             model_id = paste0(model_id, "_2")))
+d_proj = hubUtils::as_model_out_tbl(d_proj)
+
+truth_path <- system.file("truth_data.csv", package = "hubVis")
+truth_data <- read.csv(truth_path, stringsAsFactors = FALSE)
+truth_data_us <- dplyr::filter(truth_data, location == "US",
+                               time_idx < min(projection_data$target_date) + 21,
+                               time_idx > "2020-10-01")
+# Test input information
+testthat::test_that("Input parameters", {
```

Comment:
I know it's not required but I like to add it because when I am testing it locally, I don't always load the "testthat" package. I removed it to make the script more concise